modules/microshift-auto-apply-manifests.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-auto-apply-manifests_{context}"]
+= Auto-applying manifests
+
+When you run `kustomize`, {product-title} searches `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k command` to apply the manifests to the cluster. Providing multiple directories allows a flexible method of managing {product-name} workloads.
+
+[IMPORTANT]
+====
+`Kustomize` must be either running as a separate step in the boot process, or it must be part of the {product-title} image.
+====
+
+[cols="2",options="header"]
+|===
+|Location
+|Intent
+
+|`/etc/microshift/manifests`
+|Read-write location for configuration management systems or development
+
+|`/usr/lib/microshift/manifests`
+|Read-only location for embedding configuration manifests on ostree based systems
+|===
+
+== Manifest Example
+This example demonstrates automatic deployment of a busybox container using `kustomize` manifests in the `/etc/microshift/manifests` directory.
+
+.Procedure
+. Create the busybox manifest files by running the following commands:
++
+[source,terminal]
+----
+MANIFEST_DIR=/etc/microshift/manifests
+sudo mkdir -p ${MANIFEST_DIR}
+sudo tee ${MANIFEST_DIR}/busybox.yaml &>/dev/null <<EOF
```

Comment:
Commands need to be preceded with a marker (usually `$`) and can only be one per block. You may need to describe each command briefly to accomplish this.